simple-mpc-current-playlist.el

```diff
@@ -83,10 +95,32 @@ region is active, it deletes all the tracks in the region."
     (simple-mpc-call-mpc nil (list "del" (number-to-string (line-number-at-pos (point))))))
   (simple-mpc-view-current-playlist))
 
-(defun simple-mpc-maybe-refresh-playlist ()
-  "If the current buffer is a simple mpc playlist buffer, refresh its contents."
+(defun simple-mpc-maybe-refresh-playlist (&optional keep-point)
+  "If the current buffer is a simple mpc playlist buffer, refresh its contents.
+
+If optional argument KEEP-POINT is t, try to keep point in its current
+position. Otherwise, move it to the current track in the playlist."
   (when (string= (buffer-name) simple-mpc-current-playlist-buffer-name)
-    (simple-mpc-view-current-playlist)))
+    (let ((simple-mpc--paylist-keep-point-when-refreshing keep-point))
+      (simple-mpc-view-current-playlist))))
```

Comment:
I would prefer not to wrap `simple-mpc-view-current-playlist` in a let like that. It's essentially using a global variable where I don't think we really need one. Perhaps instead we could simply pass on the `keep-point` argument to `simple-mpc-view-current-playlist`. In that case we can just make the first two arguments nil so that you end up with `(simple-mpc-view-current-playlist nil nil keep-point)`. It also means we can get rid of the `simple-mpc--paylist-keep-point-when-refreshing` variable entirely.